EventFilter/L1TRawToDigi/plugins/implementations_stage2/GTCollections.h

```diff
@@ -40,6 +43,7 @@ namespace l1t {
       inline EtSumBxCollection* getZDCSums(const unsigned int copy) override { return zdcsums_[copy].get(); };
       inline JetBxCollection* getJets(const unsigned int copy) override { return jets_[copy].get(); };
       inline TauBxCollection* getTaus(const unsigned int copy) override { return taus_[copy].get(); };
+      inline float* getCICADAScore() override { return cicadaScore_.get(); };
```

Comment:
Generl question associated to the above: Is it clear that this will ever be 'just' a float, not in future calculating several values for different anomaly calo detectors, and/or for different BXs?

Reply:
```float*``` seems overkill, ```float``` would do? But ok, this pattern is used for the other products.

Reply:
> General question associated to the above line: Is it clear that this will ever be 'just' a float, not in future calculating several values for different anomaly calo detectors/algorithms, and/or for different BXs?

It's a good question actually. From the Calo summary side (the card being plugged into uGT), we don't make BX distinctions, however it is/was unclear to me whether the typical BX behavior of L1T was a product of sub-systems feeding that information to uGT, or uGT retaining BX information and the emulators shuffling that responsibility to someone else for convenience. In the first case, and it is my understanding that this is what happens, we don't need to unpack other CICADA/summary BXs, but if it were the second, we would. Again, to the best of my ability to figure out, I think we do not need to construct BX related info, or expect to see it in the uGT input record, because CICADA will not provide it.

Reply:
> General question associated to the above line: Is it clear that this will ever be 'just' a float, not in future calculating several values for different anomaly calo detectors/algorithms, and/or for different BXs?

After discussion with uGT, it is not "just a float", and modifications to the setup have been made from 49ad545fc7b871b2e01e084431ff04f0778b70c7 on. See https://github.com/cms-sw/cmssw/pull/44222#issuecomment-2054144866